test/controllers/collections_controller_test.rb

```diff
@@ -124,19 +147,52 @@ class CollectionsControllerTest < ActionController::TestCase
 
       #Since a granule is chosen at random, a full mock can not be used.
       #in this instance, we return a set collection of results for any call using this concept id and granule keyword.
```

Comment:
We should include 2 tests, 1) A test for including a random echo10 granule, 2) A test for including a random umm-granule.   Just to verify things work.